<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SprintPlanningPage</class>
 <widget class="QWidget" name="SprintPlanningPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0,0">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Sprint Planning</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="sprintScopeWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="sprintScopeLabel">
         <property name="text">
          <string>Sprint Scope</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="sprintScopeListWidget">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="implementationPlanWidget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="implementationPlanLabel">
         <property name="text">
          <string>Implementation Plan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="planTabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="planTab">
          <attribute name="title">
           <string>Implementation Plan</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextEdit" name="implementationPlanTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="advisoryTab">
          <attribute name="title">
           <string>AI Advisory Panel</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="advisoryGroupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,1,0">
              <property name="topMargin">
                  <number>5</number>
              </property>
              <property name="bottomMargin">
                  <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="advisoryInstructionLabel">
                <property name="text">
                 <string>Audit the plan to ensure the generated code will meet project specifications:</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                <widget class="QPushButton" name="runAuditButton">
                <property name="text">
                    <string>Run Audit...</string>
                </property>
                </widget>
                </item>
                <item>
                <spacer name="horizontalSpacer_2">
                <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                    <size>
                    <width>40</width>
                    <height>20</height>
                    </size>
                </property>
                </spacer>
                </item>
                </layout>
            </item>
              <item>
               <widget class="QTextEdit" name="auditResultTextEdit">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="incorporateFeedbackButton">
                  <property name="text">
                   <string>Incorporate Feedback...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="metricsLabel">
     <property name="text">
      <string>Items: 0 | Total Complexity: 0 story points | Development Tasks: 0</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="complexityLegendLabel">
     <property name="text">
      <string>&lt;b&gt;Complexity Estimation:&lt;/b&gt; Small = 1 story point, Medium = 3 story points, Large = 5 story points</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="removeFromSprintButton">
       <property name="text">
        <string>Remove from Sprint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelSprintButton">
       <property name="text">
        <string>Cancel Sprint</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="savePlanButton">
       <property name="text">
        <string>Save Plan...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startSprintButton">
       <property name="text">
        <string>Start Sprint</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>